packages/fe-mockserver-core/test/unit/__testData/RootElement.json

```diff
@@ -1,7 +1,7 @@
 [
     {
         "ID": 1,
-        "Prop1": "First Prop",
+        "Prop1": "SomethingElse",
```

Comment:
this is quite annoying, btw.: Running the tests usually results in a change to this file, even if nothing else changed.

Reply:
Yeah :) we should fix that